apps/wallet/src/state/bootstrap.ts

```diff
@@ -45,6 +45,7 @@ export const bootstrap = async (): Promise<void> => {
 	]);
 
 	configureDeviceAndNotification();
+	controlNavigationDisplay();
```

Comment:
should not trigger this one here, just call it right in side `apps/wallet/src/state/app.ts` after declaration, or make sure inital values are accurate; put this method here cause unessary complexity/confusion.. I guess.